phpcs.xml.dist

```diff
@@ -7,6 +7,8 @@
 
     <rule ref="Cdn77"/>
 
+    <rule ref="SlevomatCodingStandard.Functions.DisallowTrailingCommaInDeclaration" />
```

Comment:
Definitely against this one, I would go exactly the other way -> require the trailing slash.

It makes life much easier when moving the variables around, and it produces clearer diffs when adding/removing new variables.

Reply:
i sort of agree with @grongor , but it looks weird.

Reply:
It's one of the best "new features" in PHP IMO :D I'm really missing it in Java for example... "weird look" is just something to get used to, but otherwise it's no brainer imo.

Reply:
yeah i guess so.

Reply:
Do we want inverse PR then?

Reply:
My point was to have a rule for this, since there was none but have no preference.

Reply:
Yep, good thinking. Yes, please do the opposite of this one :)